File: desafio-02/danielmascarenhas/java/Primos.java
public class Primos {

    public static void main(String[] args) {
        
        for (int i = 2; i < 1000; i++) {
            boolean primo = true;
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    primo = false;
                    break;
                }
            }
            if (primo)
                System.out.println(i);
        }
    }
}

File: desafio-02/leohiroshi/primos.py
numero = 2
primos = []

while numero <= 10000:
    primo = True
    for i in range(2, int(numero ** 0.5) + 1):  
        if numero % i == 0:
            primo = False
            break
    if primo:
        primos.append(str(numero))  # Adiciona o número primo como string na lista
    numero += 1

# Exibe todos os primos em uma linha, separados por espaço
print(f'Os números primos de 1 a 10.000 são: {" ".join(primos)}')
